testsuite/forge/src/backend/k8s/node.rs

```diff
@@ -57,7 +60,7 @@ impl Node for K8sNode {
     }
 
     fn json_rpc_endpoint(&self) -> Url {
-        Url::from_str(&format!("http://{}:{}/v1", self.ip(), self.port())).expect("Invalid URL.")
+        Url::from_str(&format!("http://{}:{}/v1", self.dns(), self.port())).expect("Invalid URL.")
```

Comment:
does DNS work now? I assume previously it will hit some DNS translation issue

Reply:
We're assuming that `forge` is always run from within a testnet, and has access to the internal DNS. The issue with using IPs directly is that we're bypassing the k8s service abstraction, and thus the port 80 is invalid.